R/utils.R

```diff
@@ -268,15 +268,26 @@ auto_font <- function(txt, txt_max = 45, size_min = .75, size_max = 1.5,
 
 
 #' The 'Build Community plot' function
+#' @description 
+#' build_comm_plotly() is responsible for building the plotly graphic of community usage metrics
```

Comment:
This is kind of weak description, haha. Try to be more specific.

```suggestion
#' Responsible for building an interactive `{plotly}` graphic containing the trend line for number of CRAN pkg downloads by month.
```

Reply:
@AARON-CLARK Updated